src/components/ATATPhoneInput.vue

```diff
@@ -69,26 +70,31 @@
         :hide-details="true"
         :suffix="suffix"
         :prefix="this._selectedCountry.countryCode"
+        :rules="rules"
+        @blur="onBlur"
         autocomplete="off"
       >
       </v-text-field>
     </div>
+    <ATATErrorValidation :errorMessages="errorMessages"/>
```

Comment:
too much space above error message. It's caused by vuetify details div on the country dropdown.  
![image](https://user-images.githubusercontent.com/90333349/163857037-cb365fdc-15f5-4951-9bbf-c03980b16b1a.png)
You can add this if the details div needs to be there for whatever reason, or maybe `:hide-details="true"` on that dropdown (shrugs)
<img width="232" alt="image" src="https://user-images.githubusercontent.com/90333349/163857417-5bc71c54-a921-4e56-89b8-dbc42ee22293.png">